apps/server/src/api-data/osc/osc.controller.ts

```diff
@@ -1,10 +1,11 @@
-import type { ErrorResponse, OSCSettings } from 'ontime-types';
+import { type ErrorResponse, type OSCSettings } from 'ontime-types';
```

Comment:
is there a reason for this change? both expressions are the same

I am just wondering if your tooling is doing this for some reason

Reply:
yeah think it is when intelisens imports stuff for me